src/components/input/date/range/index.tsx

```diff
@@ -10,25 +11,36 @@ import { Color, Data } from "./../../../../model"
 	styleUrl: "style.scss",
 	scoped: true,
 })
-export class SmoothlyInputDateRange implements Clearable, Input {
+export class SmoothlyInputDateRange implements Clearable, Input, Editable {
 	@Element() element: HTMLElement
 	@Prop() name: string = "dateRange"
 	@Prop({ reflect: true, mutable: true }) color?: Color
 	@Prop({ reflect: true, mutable: true }) looks: Looks = "plain"
+	@Prop({ reflect: true, mutable: true }) readonly = false
 	@Prop({ reflect: true }) showLabel = true
 	@Prop({ mutable: true }) start: isoly.Date | undefined
 	@Prop({ mutable: true }) end: isoly.Date | undefined
+	@Prop() placeholder = "from - to"
```

Comment:
This should be an en dash, "–", instead of hyphen.

Reply:
Sure